pkg/converter/unstructured.go

```diff
@@ -40,21 +40,35 @@ func NestedResources(obj interface{}, resource map[string]interface{}, path ...s
 	if !found {
 		return nil, found, nil
 	}
-	valSlice, ok := val.([]interface{})
-	if !ok {
-		return nil, true, errors.New("not a slice")
-	}
-	var containers []map[string]interface{}
-	for _, item := range valSlice {
-		u, ok := item.(map[string]interface{})
-		if !ok {
-			return nil, true, errors.New("not a map")
+	valSliceOfMaps, ok := val.([]map[string]interface{})
```

Comment:
Is it possible to add a test case for the NestedResources function.

Reply:
Fixed.